site_root/input/resources/profiles/StructureDefinition-kfdrc-specimen-no-phi.json

```diff
@@ -39,7 +39,7 @@
         "sliceIsConstraining": false,
         "label": "age_at_event",
         "short": "Age time of event",
-        "definition": "Age (in days since birth) of the Participant at time of event",
+        "definition": "Age (in days since birth) of the Participant at time of event.",
```

Comment:
I don't think this profile needs to include the `age-at-event` extension since it extends the `kfdrc-specimen` profile which already includes it.

Reply:
Right, this attribute should be inherited from the base definition. Updated and Validated!